README.md

```diff
@@ -1242,6 +1244,124 @@ spec:
     emphemeral: true # VERY important. otherwise data inside the workdir and /tmp is not cleared between builds
 ```
 
+#### Docker image layers caching
+
+`docker` stores pulled and built image layers in the [daemon's (note not client)](https://docs.docker.com/get-started/overview/#docker-architecture) [local storage area](https://docs.docker.com/storage/storagedriver/#sharing-promotes-smaller-images) which is usually at `/var/lib/docker`.
+
+By leveraging RunnerSet's dynamic PV provisioning feature and your CSI driver, you can let ARC maintain a pool of PVs that are
+reused across runner pods to retain `/var/lib/docker`.
+
+_The examples below are based on the all in one runner image (`dockerdWithinRunnerContainer: true`) where the Docker daemon is running in the runner container itself with the client rather than in a sidecar:_
```

Comment:
The below example was almost intended for the case you didn't set `dockerdWithinRunnerContainer: true`. But it's confusing anyway!
How about d7fe702?